tox.ini

```diff
@@ -14,7 +14,8 @@ extras = swift
 [testenv:doc_build]
 basepython=python3
 commands =
-    {envpython} {envbindir}/sphinx-build -a -W -b html docs docs/html
+    {envpython} {envbindir}/sphinx-build -a -W --keep-going -b html docs docs/html
+    {envpython} {envbindir}/sphinx-build -a -W --keep-going -b linkcheck docs docs/html
```

Comment:
This is not lazy this is smart.

Reply:
I thought it was clear enough I was joking :sweat_smile: